packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```
@@ -1,507 +1,638 @@
-const { KeymapConfigScreen } = require('../KeymapConfig/KeymapConfigScreen');
-import SettingComponent, { UpdateSettingValueEvent } from './controls/SettingComponent';
-import shim, { MessageBoxType } from '@joplin/lib/shim';
-
+import * as React from "react";
+import Sidebar from "./Sidebar";
+import ButtonBar from "./ButtonBar";
+import Button, { ButtonLevel } from "../Button/Button";
+import { _ } from "@joplin/lib/locale";
+import bridge from "../../services/bridge";
+import Setting, {
+  AppType,
```

ESLint

[error] Expected indentation of 1 tab but found 2 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 1 tab but found 2 spaces.

(indent)